test/WebSites/ApiExplorerWebSite/ApiExplorerDataFilter.cs

```diff
@@ -9,11 +10,11 @@
 namespace ApiExplorerWebSite
 {
     /// <summary>
-    /// An action filter that looks up and serializes Api Explorer data for the action.
+    /// A resource filter that looks up and serializes Api Explorer data for the action.
     ///
     /// This replaces the 'actual' output of the action.
     /// </summary>
-    public class ApiExplorerDataFilter : ActionFilterAttribute
+    public class ApiExplorerDataFilter : IResourceFilter
```

Comment:
That's exactly the reason. No reason to actually run model binding, that just makes it hard to author the tests.